docs/2.generators/pm-install.md

```diff
@@ -30,6 +30,9 @@ The `pm-install` or `pm-i` generator generates installation commands for several
 
     # bun
     bun install -D package-name
+
+    # deno
+    deno install -D package-name
```

Comment:
Is there a way for Deno we could display `--dev` - we don't support shorthand `-D` flag.

Reply:
[`f31ce74` (#74)](https://github.com/unjs/automd/pull/74/commits/f31ce74184da5a33c9867da269cc656aba5afa96)